accepted/00060-hub-general.md

```diff
@@ -0,0 +1,109 @@
+- Feature Name: hub_general
+- Start Date: 2019-10-09
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+We want to define an Uyuni-based solution using multiple Servers to be able to manage a bigger number of clients than it is possibly doable with just one Server.
+
+Please note that this will be detailed in several upcoming RFCs. This RFC is only meant to provide context and high-level design directions, but it is not meant to be binding or technically implementable in one step.
+
+# Motivation
+[motivation]: #motivation
+
+**Scalability**: some of our users need to manage large amounts of clients (tens of thousands). Currently, this is impossible to achieve using a single Server and there is no forecast of being able to accomplish that in the foreseeable future.
+We want to define a new Uyuni-based solution to provide a reasonable subset of Uyuni functionality at that scale.
+
+**Store independence**: an additional specific problem arises in retail environments, where the need is to manage a very large number of clients in a large number of physical locations (stores). Stores should be able to use a reasonable subset of Uyuni functionality locally even in temporary or prolonged absence of a connection to the Internet or a company intranet, in particular to a central data center.
+
+## Requirements
+[Requirements]: #Requirements
+
+### Functional requirements (by importance)
+  * Hub implements visibility over the whole infrastructure (Hub → Servers → Proxies → clients)
+  * Hub manages Servers in terms of upgrades
+  * Hub implements API endpoints to act on Servers
+    * Main expected use cases:
+      * Content management (eg. channel cloning)
+      * Automated patching
+      * Configuration management
+  * Hub implements aggregated reporting
+    * client data to be pulled from Servers periodically
+  * Hub implements centralized content management of (in order of importance)
+    * Packaged software
+    * Configuration Management data (states, pillars...)
+    * OS Images
+    * Container Images
+  * Managed content can be selectively synced from Hub to Servers
+    * Same order of importance defined above
+    * Synchronization has to be atomic
```

Comment:
This means that Servers will not be left in a state where content is only partially synced, as it could be inconsistent (imagine Salt states being synced while packages are not).

This does not necessarily mean that any downloaded byte will have to be re-downloaded.

`spacewalk-repo-sync` is by default not atomic.

Do you think a text change is needed here?